### ISSUE TITLE
[!!] First sentence

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-selecting-eval-tools#15]
First sentence:
> Web accessibility evaluation tools are software programs or online services that help determine if a website is accessible.

Issues: "determine if a website is accessible" sounds like it's something that you do at the end, whereas we encourage evaluation early and throughout development. Often tools are primarily used to identify potential accessibility barriers, not just check if it's accessible at the end. Therefore, this sentence could be made more precise and inclusive of different uses.

Suggest something like:
> Web accessibility evaluation tools are software programs or online services that help determine if a website meets accessibility standards and guidelines.

Or even more precise:
> Web accessibility evaluation tools are software programs or online services that help determine what accessibility requirements a website meets or does not meet.

I'd like some input from others in EOWG. If people are generally OK with it as is, then I'm OK letting it go for now.

**Comment by shawna-slh** [comment]
I just noted that the Tools List starts with:

> Web accessibility evaluation tools are software programs or online services that help you determine if web content meets accessibility guidelines.

**Comment by vavroom** [comment]
Happy enough to align with the Tools List.
